ios/Classes/PdftronFlutterPlugin.m

```diff
@@ -2708,6 +2722,76 @@ - (void)openLayersList:(FlutterResult)flutterResult
     flutterResult(nil);
 }
 
+-(void)openThumbnailsView:(FlutterResult)flutterResult
+{
+    PTDocumentController *documentController = [self getDocumentController];
+    if (documentController) {
+        [documentController showThumbnailsController];
+    }
+
+    flutterResult(nil);
```

Comment:
If there is no `documentController`, should we return something else (an error)?